### PR TITLE
[iOS] Added iOS Nightly

### LINKED ISSUE #5741: [iOS] Nightly package feed
Laundry list.
1. remove .swiftpm from .gitignore
2. generate tags for nightly
3. limit what's included or shown in the package
4. add umbrella header for module map generation

### PR DESCRIPTION
**Pull request by jwoo-msft**
# Related Issue
Fixed #5741 

# Description
* added umbrella header for module generation
* added resources for images & XIB
* updated bundle interfaces
* Swift Package Manager allows taking a dependency via any of the below options
  1. git tags (versions, v1.5)
  2. branches (main)
  3. commits
 by choosing the branch option and selecting `main`, one can CI our builds.
# How Verified
Created a sample project to test the package.
Tested it with the existing sample app

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/5805)

### CODE REVIEW

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputChoiceSetRenderer.mm [hunk 1]
```diff
@@ -38,11 +38,8 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
     std::shared_ptr<BaseCardElement> elem = [acoElem element];
     std::shared_ptr<ChoiceSetInput> choiceSet = std::dynamic_pointer_cast<ChoiceSetInput>(elem);
     // creates a tableview with pre-defined style
-    NSBundle *bundle = [NSBundle bundleWithIdentifier:@"MSFT.AdaptiveCards"];
-    if (!bundle) { // https://github.com/Microsoft/AdaptiveCards/issues/1834
-        return nil;
-    }
-    ACRInputTableView *choiceSetView = [bundle loadNibNamed:@"ACRInputTableView" owner:self options:nil][0];
+    //    ACRInputTableView *choiceSetView = [SWIFTPM_MODULE_BUNDLE loadNibNamed:@"ACRInputTableView" owner:self options:nil][0];
```
Comment (paulcam206) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputChoiceSetRenderer.mm:41]: do we need this comment? :)

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/include/ACOAdaptiveCard.h [hunk 2]
```diff
@@ -0,0 +1 @@
+../ACOAdaptiveCard.h
```
Comment (paulcam206) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/include/ACOAdaptiveCard.h:1]: just curious here -- is there no cleaner way of doing this?

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputChoiceSetRenderer.mm (hunk 1 above)
Comment (jwoo-msft) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputChoiceSetRenderer.mm:41]: thanks, @paulcam206 I left a comment for removing this comment in my first pass-through, and forgot to submit my review of the first pass.

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/include/ACOAdaptiveCard.h (hunk 2 above)
Comment (jwoo-msft) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/include/ACOAdaptiveCard.h:1]: Unfortunately, I couldn't find one.

**Pull request merged**